priv/perf/apps/load_test/lib/runner/watcher_info.ex

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.Runner.WatcherInfo do
+  @moduledoc """
+  Load test for watcher info
+  """
+  use Chaperon.LoadTest
+
+  @concurrent_sessions 1
+  @iterations 1
+
+  def default_config(),
+    do: %{
+      merge_scenario_sessions: true
+    }
+
+  def scenarios(),
```

Comment:
not to be overly pedantic, but @pgebal has broken these down into multiline functions. and @T-Dnzt likes that too:
```
def scenarios() do
    {:ok, sender} = Account.new()

    [
      {{@concurrent_session, [LoadTest.Scenario.CreateUtxos, LoadTest.Scenario.SpendUtxos]},
       %{
         sender: sender
       }}
    ]
  end
```

Reply:
Are multiline functions generally preferred? I don't see any recommendation in the style guide.
(although I imagine sometimes a function is  obviously a one liner)

Reply:
yeah exactly, if the function doesn't span through multiple lines, a one liner can be used.